brut.apktool/apktool-cli/build.gradle.kts

```diff
@@ -20,6 +21,7 @@ buildscript {
 
 dependencies {
     implementation("commons-cli:commons-cli:$commonsCliVersion")
+    implementation("com.android.tools.smali:smali-baksmali:$baksmaliVersion")
```

Comment:
I don't see any baksmali used in apktool-cli?

Reply:
this is needed because I extended BaksmaliOptions in Config, which is called inside of Main in apktool-cli